Blog post for the jdk25+36 release

adamfarley opened the pull request:
# Description of change

Blog post for the JDK25+36 release.

## Checklist

<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] `npm test` and `npm run build` passes
- [ ] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/main/CONTRIBUTING.md)

Resolves https://github.com/adoptium/adoptium.net/issues/308

adamfarley committed:
Blog post for the jdk25+36 release

Signed-off-by: Adam Farley <adfarley@ibm.com>

jerboaa reviewed (approved):
Looks OK to me (the jmods part).

adamfarley committed:
Add link to windows aarch64 issue

Signed-off-by: Adam Farley <adfarley@ibm.com>

adamfarley committed:
Updating blog date

Signed-off-by: Adam Farley <adfarley@ibm.com>

adamfarley committed:
Amending the Windows build entry

Signed-off-by: Adam Farley <adfarley@ibm.com>

adamfarley commented:
One more review and a merge, please.

adamfarley committed:
Removing entry for JDK25 Aarch64 Windows

Because we are not certain what will happen, so adding a "maybe"
has been deemed by the community to be too uncertain for a release
blog.

Signed-off-by: Adam Farley <adfarley@ibm.com>

adamfarley committed:
Merge branch 'main' into jdk25_release_blog

adamfarley commented:
Thanks all 🤓
